app/services/atmosphere/cloud/scale_appliance.rb

```diff
@@ -0,0 +1,51 @@
+module Atmosphere
```

Comment:
Please provide a short code documentation. One things that would be useful is to describe shortly how does the `quantity` parameter influence the algorithm.